cmd/azure-connector/config.go

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2022 Contributors to the Eclipse Foundation
+//
+// See the NOTICE file(s) distributed with this work for additional
+// information regarding copyright ownership.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License 2.0 which is available at
+// https://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+
+package main
+
+import (
+	"flag"
+
+	azurecfg "github.com/eclipse-kanto/azure-connector/config"
+)
+
+// AzureSettingsExt extends the general configurable data of the azure Connector with configuration for the message handlers.
+type AzureSettingsExt struct {
+	*azurecfg.AzureSettings
+	PassthroughCommandTopic    string `json:"passthroughCommandTopic"`
+	PassthroughTelemetryTopics string `json:"passthroughTelemetryTopics"`
+}
+
+func defaultAzureSettingsExt() *AzureSettingsExt {
+	azureSettings := azurecfg.DefaultSettings()
+	return &AzureSettingsExt{
+		PassthroughCommandTopic:    "cloud-to-device",
+		PassthroughTelemetryTopics: "device-to-cloud",
+		AzureSettings:              azureSettings,
+	}
+}
+
+func addMessageHandlerFlags(f *flag.FlagSet, settings *AzureSettingsExt) {
```

Comment:
Cyclic reference between config and flags package

Reply:
no cyclic reference here, config.go file is in main package.